catalog.yaml

```diff
@@ -246,6 +246,12 @@ topics:
         name: Terraform_Kubernetes_AWS_EKS
         path: kubernetes/terraform/gke
         description: Create an AWS Kubernetes using Terraform automation
+      - id: spot_ocean
+        name: SpotOcean optimized cluster
+        path: kubernetes/spot
+        description: Create a Spot Ocean Kubernetes cost optimized Cluster
+        image_url: kubernetes/spot/spot.png
```

Comment:
rename the file to "logo.png" and remove the line, it will be generated automatically